product_docs/docs/transporter/ba_preview/getting_started/create_database.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Creating a database cluster"
+---
+
+Before you migrate your database to EDB Postgres® AI, create an empty database in a database cluster. 
+You will configure this database to be the target of your migration when using EDB Transporter. 
+
+1.  Access the [EDB Postgres AI Console](https://portal.biganimal.com) and log in with your EDB Postgres AI Database Cloud Service credentials.
+
+1.  Select the project where you want to create the database cluster. 
+
+    See [Creating a project](/biganimal/latest/administering_cluster/projects/#creating-a-project) if you want to create one.
+
+1.  Within your project, select **Create New** and **Database Cluser** to create an instance that will serve as target for the EDB Transporter migration.
+
+    See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/) for detailed instructions on how to create a single-node or a primary/standby high availability cluster. 
+
+    See [Creating a distributed high-availability cluster](/biganimal/latest/getting_started/creating_a_cluster/creating_a_dha_cluster/) for detailed instructions on how to create a distributed high availibility cluster.
+
+1.  In **Clusters** page, select your cluster, and use the **Quick Connect** option to access your instance from your terminal. 
+
+1.  Create a new empty database. For an example, see [Create a new database](/biganimal/latest/free_trial/quickstart/#create-a-new-database).
```

Comment:
IF PGD/DHA clusters are supported, they come with a default database. We would need to differentiate here, e.g.: 

Ensure there is an empty database in your server.
  For single-node or primary/standby high availability clusters, you must create a database and leave it empty.
  For distributed high availability clusters, use the default database that is created by the system. Other manually created databases won’t be replicated.